Automattic/jetpack
# [Plugin] Backup: Update/upgrade button link

Update upgrade button link to point to the checkout page.

#### Changes proposed in this Pull Request:
Currently, the Upgrade button on the Jetpack Backup plugin points to:
https://jetpack.com/upgrade/backup/?site=[your-site], where the user can then click on `Back up your site` to pay for Jetpack Backup. 
This PR changes the target of the button and it now points directly to the checkout page. It is currently adding the Jetpack Backup Daily plan. 

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1- On a new site, add the Jetpack Backup plugin
2- go to the plugin admin page on your site and click on the upgrade button
3- Confirm that you're taken directly to the checkout page and that Jetpack Backup Daily is added to the cart.
